site/app/templates/admin/SystemUpdate.twig

```diff
@@ -0,0 +1,43 @@
+<div class="content">
+    <h2>System Update</h2>
+    <h4> Current Version Installed {{ latest_tag }} </h4>
+    <a target="_blank" href="https://submitty.org/sysadmin/update">Need help with updating Submitty? <i style="font-style:normal;" class="fa-question-circle"></i></a>
+    <button class="btn btn-primary" onclick="getReleases('{{latest_tag}}');">Check for Updates</button>
+    <div class="content" id="update_status" hidden>
+        <h2 id="text"></label>
+        <h4 id="tag"></label> 
+    </div>
+
+    <div class="content" id="release_notes_container" hidden>
+        <label id="release_notes"></label>
+    </div>
+</div>
+
+
+<script>
```

Comment:
This should not be inline. Move into an existing js file or its own.

Reply:
After moving it would I need $this->core->getOutput()->addInternalJs in homePageController, since there isnt a php file for the new page at the moment